src/reporter.ts

```diff
@@ -32,6 +32,7 @@ export default class SauceReporter extends SummaryFormatter {
   cucumberVersion: string;
   startedAt?: string;
   endedAt?: string;
+  suiteEndAt?: Date;
```

Comment:
1. Dangerous duplicity in naming ⚠️ The SauceReporter already has a field called `suiteName`, suggesting that the entire thing here is relevant to this one "suite". If we continue this line of thought, that means `startedAt` and `endedAt` relate to `suitName`. So if `endedAt` represents the end date for the suite, what is `suiteEndAt`?
2. When introducing fields like these, it's imperative to keep consistency with neighbors. `endedAt`, `startedAt`. Therefore `suiteEndAt` ought to be called `suitedEndedAt`.

Reply:
I can't really recall why I set the `suiteName`. But the `suiteName` indeed need to be improved. It's more like a job name or report name actually, not a suite. :-(

This time I'd like to set the suite end time as a local variable.